src/main/cpp/Constants.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Constants.h"
+
+namespace RobotConstants {
+    // although I think getPosition() might return in native units of
+    // rotations so this would then be 1
+    const double neo550TicksPerRev = 42;
+
+    const double neoTicksPerRev = 42;
+    const double falconTicksPerRev = 2048;
+
+    // if enabled, disables a lot of the automatic shuffleboard values
+    const bool competition = false;
+} // namespace RobotConstants
+
+namespace DrivetrainConstants {
+    const double ticksPerRev = RobotConstants::neoTicksPerRev;
+    const double gearRatio = 1 / 9.64;
+    const double wheelDiameter = 6;
```

Comment:
should we add a comment that gives units for wheel size? I know it's inches, but does anyone else?

Reply:
Yeah, I added this in the TOML update